### PR TITLE
fix: type all components

### PR DESCRIPTION
**Pull request by huydosgtech**
## Check List
- [ ] Your code builds clean without any errors or warnings.
- [ ] Check coding convention.
- [ ] Updated Storybook components and stories to reflect any changes made to the UI.
- [ ] Test covers all edge cases and coverage is greater than or equal to 80%.

## Proof of Completeness
None

### CODE REVIEW

**Review by loc-nguyenthien — APPROVED** [review]

File: src/components/Card/Card.tsx [hunk 1]
```diff
@@ -26,7 +26,7 @@ const Card: React.FC<CardProps> = ({onPress, style, children}) => (
   </CardWrapper>
 )
 
-const CardWrapper = styled(TouchableOpacity)`
+const CardWrapper = styled.TouchableOpacity`
```
Comment (baonguyenhsts) [inline src/components/Card/Card.tsx:29]: We should remove the ``

File: src/components/Slider/Slider.tsx [hunk 2]
```diff
@@ -311,7 +312,14 @@ const Slider: SliderComponentProps = ({
             onPressPoint={(point: number) => tapToSeek && onPressPoint(point)}
           />
         )}
-        <Tracked style={[trackedStyle, animatedTrackStyle]} />
+        <Track
+          style={StyleSheet.flatten([
+            styles.track,
```
Comment (baonguyenhsts) [inline src/components/Slider/Slider.tsx:317]: The `styles.track` is StyleSheet.absoluteFillObject so we do not need create this style

File: src/components/Slider/SliderFixed.tsx [hunk 3]
```diff
@@ -183,7 +185,14 @@ const SliderFixed: React.FC<SliderProps> = ({
             onPressPoint={(point: number) => tapToSeek && onPressPoint(point)}
           />
         )}
-        <Tracked style={[trackedStyle, animatedTrackStyle]} />
+        <Track
+          style={StyleSheet.flatten([
+            styles.track,
```
Comment (baonguyenhsts) [inline src/components/Slider/SliderFixed.tsx:190]: The styles.track is StyleSheet.absoluteFillObject so we do not need create this style

File: src/components/Slider/SliderFixedRange.tsx [hunk 4]
```diff
@@ -267,7 +266,14 @@ const SliderFixedRange: React.FC<SliderFixedRangeProps> = ({
             onPressPoint={(point: number) => tapToSeek && onPressPoint(point)}
           />
         )}
-        <Tracked style={[trackedStyle, animatedTrackStyle]} />
+        <Track
+          style={StyleSheet.flatten([
+            styles.track,
```
Comment (baonguyenhsts) [inline src/components/Slider/SliderFixedRange.tsx:271]: The styles.track is StyleSheet.absoluteFillObject so we do not need create this style

File: src/components/Slider/SliderRange.tsx [hunk 5]
```diff
@@ -329,7 +328,14 @@ const SliderRange: React.FC<SliderRangeProps> = ({
             onPressPoint={(point: number) => tapToSeek && onPressPoint(point)}
           />
         )}
-        <Tracked style={[trackedStyle, animatedTrackStyle]} />
+        <Track
+          style={StyleSheet.flatten([
+            styles.track,
```
Comment (baonguyenhsts) [inline src/components/Slider/SliderRange.tsx:333]: The `styles.track` is StyleSheet.absoluteFillObject so we do not need create this style

File: src/components/Slider/SliderFixedRange.tsx (hunk 4 above)
Comment (baonguyenhsts) [inline src/components/Slider/SliderFixedRange.tsx:271]: The `styles.track` is StyleSheet.absoluteFillObject so we do not need create this style

File: src/components/Slider/SliderFixed.tsx (hunk 3 above)
Comment (baonguyenhsts) [inline src/components/Slider/SliderFixed.tsx:190]: The `styles.track` is StyleSheet.absoluteFillObject so we do not need create this style

File: src/components/Slider/Slider.tsx (hunk 2 above)
Comment (baonguyenhsts) [inline src/components/Slider/Slider.tsx:317]: The `styles.track` is StyleSheet.absoluteFillObject so we do not need create this style

**Review by baonguyenhsts — CHANGES_REQUESTED** [review]

File: src/components/Card/Card.tsx (hunk 1 above)
Comment (loc-nguyenthien) [reply src/components/Card/Card.tsx]: Updated

File: src/components/Slider/SliderRange.tsx (hunk 5 above)
Comment (loc-nguyenthien) [reply src/components/Slider/SliderRange.tsx]: Updated

File: src/components/Slider/SliderFixedRange.tsx (hunk 4 above)
Comment (loc-nguyenthien) [reply src/components/Slider/SliderFixedRange.tsx]: Updated

File: src/components/Slider/SliderFixed.tsx (hunk 3 above)
Comment (loc-nguyenthien) [reply src/components/Slider/SliderFixed.tsx]: Updated

File: src/components/Slider/Slider.tsx (hunk 2 above)
Comment (loc-nguyenthien) [reply src/components/Slider/Slider.tsx]: Updated

**Review by baonguyenhsts — APPROVED** [review]

File: src/components/RadioButton/RadioButton.tsx [hunk 6]
```diff
@@ -225,14 +215,23 @@ const RadioButtonInnerContainer = styled(View)<{
...
 
-const LabelText = styled(Text)((props: {theme: ITheme}) => ({
+const LabelText = styled.Text((props: {theme: ITheme}) => ({
   color: props?.theme?.colors?.black,
   fontSize: props?.theme?.fontSizes?.md,
 }))
+
+const styles = StyleSheet.create({
+  bounceStyle: {
+    borderWidth: metrics.borderRadius,
```
Comment (ngochuyduong) [inline src/components/RadioButton/RadioButton.tsx:232]: Please use value from theme

Comment (loc-nguyenthien) [reply src/components/RadioButton/RadioButton.tsx]: Updated

File: src/components/Card/Card.tsx [hunk 7]
```diff
@@ -26,11 +26,10 @@ const Card: React.FC<CardProps> = ({onPress, style, children}) => (
...
 
-const CardWrapper = styled(TouchableOpacity)`
-  ${(props: {theme: ITheme}) => ({
-    padding: metrics.small,
-    borderRadius: metrics.borderRadius,
-    backgroundColor: props?.theme?.colors?.cardBackground,
-  })}
-`
+const CardWrapper = styled.TouchableOpacity((props: {theme: ITheme}) => ({
+  padding: metrics.small,
```
Comment (ngochuyduong) [inline src/components/Card/Card.tsx:30]: Same here

**Review by chungnguyenhoang — APPROVED** [review]

File: src/components/Button/Button.tsx [hunk 8]
```diff
@@ -57,7 +56,7 @@ const Button: React.FC<ButtonProps> = ({
   </ButtonWrapper>
 )
 
-const ButtonWrapper = styled(TouchableOpacity)<Omit<ButtonProps, 'text' | 'onPress'>>(
+const ButtonWrapper = styled.TouchableOpacity<Omit<ButtonProps, 'text' | 'onPress'>>(
   ({theme, backgroundColor, outline, outlineWidth, outlineColor, borderRadius, disabled}) => ({
     paddingVertical: metrics.xxs,
```
Comment (ngochuyduong) [inline src/components/Button/Button.tsx:61]: Same here

File: src/components/CodeInput/CodeInput.tsx [hunk 9]
```diff
@@ -142,7 +143,7 @@ export default memo(CodeInput)
 
 const CodeInputContainer = styled.View({})
 
-const Cell = styled(Pressable)((props: CellStyle) => ({
+const Cell = styled.Pressable((props: CellStyle) => ({
   width: metrics.huge,
```
Comment (ngochuyduong) [inline src/components/CodeInput/CodeInput.tsx:147]: Same here

File: src/components/CodeInput/CodeInput.tsx [hunk 10]
```diff
@@ -153,7 +154,7 @@ const Cell = styled(Pressable)((props: CellStyle) => ({
   margin: metrics.tiny,
 }))
 
-const SecureView = styled(Pressable)((props: SecureViewStyle) => ({
+const SecureView = styled.Pressable((props: SecureViewStyle) => ({
   width: metrics.small,
```
Comment (ngochuyduong) [inline src/components/CodeInput/CodeInput.tsx:158]: same here

File: src/components/Card/Card.tsx (hunk 7 above)
Comment (loc-nguyenthien) [reply src/components/Card/Card.tsx]: Updated

File: src/components/Button/Button.tsx (hunk 8 above)
Comment (loc-nguyenthien) [reply src/components/Button/Button.tsx]: Updated

File: src/components/CodeInput/CodeInput.tsx (hunk 9 above)
Comment (loc-nguyenthien) [reply src/components/CodeInput/CodeInput.tsx]: Updated

File: src/components/CodeInput/CodeInput.tsx (hunk 10 above)
Comment (loc-nguyenthien) [reply src/components/CodeInput/CodeInput.tsx]: Updated

**Review by honghoangsts — APPROVED** [review]

**Pull request merged**